recipe/meta.yaml

```diff
@@ -44,13 +44,15 @@ requirements:
   host:
     - {{ mpi }}  # [mpi != 'nompi']
     - libpng
-    - hdf5 * {{ mpi_prefix }}_*
+    - hdf5 1.10.5 {{ mpi_prefix }}_*
```

Comment:
FWIW this is also what we [pin globally]( https://github.com/conda-forge/conda-forge-pinning-feedstock/blob/16dc51afbb57d4604df3649ef79d0ba75ddec62d/recipe/conda_build_config.yaml#L433-L434 ). Could you please provide some context about why that doesn't work here?

Reply:
We have some other packages that depend on this one and that are explicitly fixed to 1.10.5. We had some conflicts otherwise. I'd like to get an updated version of this package with the same constraint for compatibility, and then I plan on making another pass over all the affected recipes that removes the constraint.

Reply:
cc-ing @conda-forge/hdf5 , in case others have thoughts on how to approach this

Reply:
Also worth noting more recent changes ( https://github.com/conda-forge/h5utils-feedstock/pull/25 ).